JavaScriptSPA/ui.js

```diff
@@ -0,0 +1,27 @@
+// updates the UI post login/token acquisition
+function updateUI() {
+    const userName = myMSALObj.getAccount().name;
+    logMessage("User '" + userName + "' logged-in");
+
+    // add the logout button
+    const signInButton = document.getElementById('SignIn');
```

Comment:
Instead of manipulating an existing button, I think it would be cleaner to have multiple buttons that are hidden/shown conditionally. Would clean up a bit of this code.